source/_integrations/matter.markdown

```diff
@@ -23,182 +23,241 @@ ha_platforms:
 
-At this time there are only a few devices available that are compatible with the standard and some of them require you to join a beta/developer program. It is to be expected that more devices will hit the market during the 2nd quarter of 2023 and beyond.
+Unlike the radio based protocols we're already familiar with in the IoT landscape, like Zigbee and Z-Wave, Matter uses standard IP-based communication. Matter is not a radio protocol, but a control protocol that runs on top of the existing network infrastructure, using your existing Wi-Fi/Ethernet routers and Thread.
 
-<p class='note'>
-Both the Matter standard itself and its implementation within Home Assistant are in a very early stage where only the basics are working and/or breaking changes can still happen. Using it in production is not recommended yet until it matures a bit more.
-</p>
+Home Assistant is a so-called _controller_, it can control Matter based devices. Other examples of Matter controllers are the Google Nest speakers, Apple HomePods, and a SmartThings Station.
 
-One of the great features of Matter is the so called _Multi Fabric_ feature: you can join the same device to multiple controllers. For example: add it to Google Home, Apple Home, and Home Assistant at the same time.
+## Bridge devices
 
-For devices where Home Assistant provides a native integration (with local API), Matter may not be the best option. Matter, being a universal standard, might not have the nitty-gritty features that come with a product specific protocol. A good example is Philips Hue:  the communication over Matter only provides the basic controls over lights, the official integration brings all Hue unique features like (dynamic) scenes, entertainment mode, etc.
+One of the great things about Matter is that you can have both Wi-Fi and Thread based devices on the same controller.
+Next to actual devices (like actors or sensors), you will also see bridges. The bridge connects the network over Ethernet or Wi-Fi and bridges multiple devices into a Matter network. A great example is the Philips Hue V2 bridge, which is a Zigbee hub and a Matter bridge. This bridge exposes all Zigbee devices already connected to the bridge as Matter devices on the network. Also, Aqara, SwitchBot, and IKEA have launched such Hub devices.
 
-<p class='note'>
-The Matter protocol relies on (local) IPv6 and mDNS (multicast traffic) which should be able to travel freely in your network. Matter devices (and any Thread Border routers) must be on the same LAN/VLAN as Home Assistant. Implementations like mDNS reflectors usually do more harm than good.
-</p>
+<div class='note'>
+Home Assistant, as a Matter controller, only supports **control** of Matter devices. Home Assistant is not a bridge itself and it cannot turn existing devices within Home Assistant into Matter compatible devices.
+</div>
+
+## Thread
```

Comment:
```suggestion
## Matter over Thread
```

Reply:
Can we leave it like this to draw attention to Thread ?

Reply:
If I understood correctly we should use `{% term Thread %}` instead of just `Thread`, am I correct  ?
Also I agree with @agners  about the phrasing, it is clearer for the reader what roles play Thread vs Matter when it's explicitly mentioned.
```suggestion
## Matter over {% term Thread %}
```

Reply:
I don't agree, except when Thread stands out as a world on its own, like you are proposing, in that case it is perfect.
But... I have no idea if we may use that tooltip inside a header.

Reply:
@Vinalti  Headers may be linkable. We should not add terminology pop-ups to that.

Reply:
going to leave it as-is